### DISCUSSION TITLE
how to change the ON/OF text above the button

### DISCUSSION THREAD

**Discussion by PetrJakes** [arendst/Tasmota#13333] [Q&A] [answered]
Is there a way to change the ON OFF text above buttons with a custom text?

**Comment by barbudor** [comment, marked as answer]
https://tasmota.github.io/docs/Commands/#webbutton

**Comment by PetrJakes** [comment]
The command Webbutton changes the text on the button (default text is
"Toggle"), not the text above the button (default text is "ON/OFF")

**Reply by barbudor** [reply]
https://tasmota.github.io/docs/Commands/#statetext
But that will also change the value transmitted in MQTT
There is no way to decoupple the state displayed on Web and the state transmitted in MQTT

**Comment by PetrJakes** [comment]
OMG, thanks.
I was reading the commands over and over again but I did not see it.
Txs ....
I understand about the MQTT. It is easy to catch it in MQTT (yes, it will
be a little "inconsistent" but easily doable
